portal/tests/test_captcha_forms.py

```diff
@@ -0,0 +1,66 @@
+# -*- coding: utf-8 -*-
+# Code for Life
+#
+# Copyright (C) 2017, Ocado Innovation Limited
```

Comment:
2018

Reply:
I've updated the year on all the files.

(⚠️ Out of scope) does the copyright header still work if we remove the year completely?

Reply:
legal team had advised us to state the newest year - so we should  update all our headers to 2018 (out of scope for this PR)